### ISSUE TITLE
What happened to the ability to move and resize the elements?

### ISSUE THREAD

**Issue by NovaViper** [jneilliii/OctoPrint-ConsolidatedTabs#19]
Hey, I just installed v0.0.8 and I noticed that the ability to move the tabs around is completely gone, everything is just lined in a colnum. Im using Octoprint v1.4.2 and using the following plugins,

```
Bed Visualizer (0.1.14)
DisplayLayerProgress Plugin (1.23.0)
DryRun Plugin (1.2.0)
Exclude Region (0.2.0)
FileManager (0.1.4), Macro (0.2.0)
Marlin Binary File Transfer: Protocol Mk II (0.1.4)
Navbar Temperature Plugin (0.14)
OctoPrint-WideScreen (0.1.3) 
PrintJobHistory (1.7.0) 
Printoid Plugin (1.0.2)
PrintTimeGenius Plugin (2.2.5)
PrusaSlicer Thumbnails (0.1.2)
Smart Preheat (0.0.5) 
SpoolManager Plugin (1.0.0rc4)
```
![image](https://user-images.githubusercontent.com/7191115/90669746-b9598b80-e217-11ea-92ca-73922ece6e32.png)

**Comment by jneilliii** [comment]
Documented on the release notes and readme, gold down shift and click a panel to activate and allow moving/ resizing.

**Comment by jneilliii** [comment]
just released 0.0.9 with more description in the settings on activating a tab.

**Comment by NovaViper** [comment]
Thanks!
